lua/wikis/commons/MatchTicker/DisplayComponents/New.lua

```diff
@@ -215,33 +217,23 @@ function Details:tournament()
 
 	local match = self.match
 
-	local icon = LeagueIcon.display{
-		icon = Logic.emptyOr(match.icon, TOURNAMENT_DEFAULT_ICON),
-		iconDark = match.icondark,
-		link = match.pagename,
-		name = match.tournament,
-		options = {noTemplate = true},
+	return HtmlWidgets.Div{
+		classes = {'match-tournament'},
+		children = {
+			Title{
+				tournament = {
+					pageName = match.pagename,
+					displayName = match.tournament,
+					tickerName = match.tickername,
+					icon = match.icon,
+					iconDark = match.icondark,
+					series = match.series,
+					game = match.game,
+				},
+				displayGameIcon = self.displayGameIcons
+			}
+		}
 	}
-
-	local displayName = Logic.emptyOr(
-		match.tickername,
-		match.tournament,
-		match.parent:gsub('_', ' ')
-	)
```

Comment:
Would we want to preserve this fallback logic?

Reply:
Could be a tournament missing a ticker name right? So I guess?

Reply:
yeah i think we should keep the fallback